.github/scripts/mega-app-build.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# Default values
+BUILD_TOOL="cra"
+BUILD_TOOL_VERSION="latest"
+LANGUAGE="ts"
+MEGA_APP_NAME=""
+FRAMEWORK="react"
+FRAMEWORK_VERSION="latest"
+PLATFORM="android"
+LOG_FILE="test.log"
+
+# Options
+# e.g.
+# $ ./mega-app-build.sh --build-tool react --build-tool-version latest --language typescript --name react-latest-cra-latest-node-18-ts --framework cra --framework-version latest --pkg-manager npm
+# $ ./mega-app-build.sh -bt react -btr latest -lang typescript -n react-latest-cra-latest-node-18-ts -fw cra -fwv latest -pm npm
+# $ ./mega-app-build.sh -n react-latest-cra-latest-node-18-ts
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+    -bt | --build-tool)
+        BUILD_TOOL=$2
+        shift
+        ;;
+    -btr | --build-tool-version)
+        BUILD_TOOL_VERSION=$2
+        shift
+        ;;
+    -lang | --language)
+        LANGUAGE=$2
+        shift
+        ;;
+    -n | --name)
+        MEGA_APP_NAME=$2
+        shift
+        ;;
+    -fw | --framework)
+        FRAMEWORK=$2
+        shift
+        ;;
+    -fwv | --framework-version)
+        FRAMEWORK_VERSION=$2
+        shift
+        ;;
+    -pm | --pkg-manager)
+        PKG_MANAGER=$2
+        shift
+        ;;
+    -p | --platform)
+        PLATFORM=$2
+        shift
+        ;;
+    -h | --help)
+        echo "Usage: mega-app-create-app.sh [OPTIONS]"
+        echo "Options:"
+        echo "  -bt, --build-tool            Specify the build tool: cra, next, vite, angular-cli, vue-cli, nuxt, react-native-cli, expo. (default: cra)"
```

Comment:
Don't feel strongly, but what do you think about 
(1) aligning the short hand flag (e.g. `-bt`) to be single alphabet (I think this is more conventional)
(2) or drop short hand flag to use long hand flag (`--build-tool`)?

Reply:
Done 65ad644433585dcf50f585f50b1e146035ff27a2